src/app/utils/firebase.js

```diff
@@ -26,39 +26,42 @@ const FireBaseTools = {
         default:
             throw new Error('Provider is not supported!!!');
         }
-
     },
 
 
-    getUserCourses :(dispatch, TYPE)=>{
-        var userCur=null,id = firebaseAuth.currentUser? firebaseAuth.currentUser.uid: null
-         usersRef.child(id.toString()).on('value', function (snap) {
-            if(snap.val())
-                userCur=(snap.val().coursearray);
-            if(id && !userCur) {userCur=['No Courses']}
+    getUserCourses: (dispatch, TYPE) => {
+        let userCur = null;
+        const id = firebaseAuth.currentUser ? firebaseAuth.currentUser.uid : null;
+        /* eslint-disable */
+        usersRef.child(id.toString()).on('value', function(snap) {
+            if (snap.val()) {
+                userCur = (snap.val().coursearray);
+            }
+            if (id && !userCur) { userCur = ['No Courses']; }
 
             // By not returning anything and dispatching from here,
             // the action will be dispatched every time the coursearray changes
             dispatch({
-              type: TYPE,
-              payload: userCur
+                type: TYPE,
+                payload: userCur,
             });
-
-         });
+        });
+        /* eslint-enable */
```

Comment:
If you look at line 36, Travis doesn't like where it says function(snap). Ideally, it would like to have an arrow callback.